packages/core/src/Primitive/useRender.ts

```
@@ -1,38 +1,17 @@
 import type { Component, ComponentPublicInstance, ComputedRef, MaybeRefOrGetter, Ref } from 'vue'
 import type { AsTag } from './Primitive'
+import type { PartState, StateAttributesMapping } from '../shared/stateToDataAttrs'
```

ESLint

[error] Expected "../shared/stateToDataAttrs" to come before "./Primitive".

(perfectionist/sort-imports)